scripts/compose.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker compose -f dev-compose.yaml $@
```

Comment:
Try to play with the `"$(dirname "$(readlink -f "$0")")/dev-compose.yaml` form of the file location hardcoding. This might remove the requirement to run the scripts from the repo root, but it needs to be tested (and probably the command update might be needed)

Reply:
Done. Thank you.